.github/workflows/bench-turbopack-scheduled.yml

```diff
@@ -110,32 +116,33 @@ jobs:
           name: ${{ matrix.bench.name }}
           path: raw.json
 
-      # This avoids putting this data into the rust-cache
-      - name: Clear benchmarks
-        run: rm -rf target/criterion
-
```

Comment:
I think this can't be removed, as rust-cache would include this directory into the cache.

If you can to remove the duplicate call, it should be safe to remove the first call in line 85 assuming that this directory is never stored in first place.

I think the cache is not stored in case of errors anyway, but just to be safe you could also add `if: always()`

Reply:
We are no longer caching here, just restoring cache